pkg/token/execCredentialPlugin.go

```diff
@@ -28,6 +32,15 @@ type execCredentialPlugin struct {
 }
 
 func New(o *Options) (ExecCredentialPlugin, error) {
+	env := os.Getenv(execInfoEnv)
+	//fmt.Fprintln(os.Stderr, os.Getenv(execInfoEnv))
```

Comment:
please remove unnecessary commented out code. for debugging purpose, use klog with higher verbosity

Reply:
Done for next commit. Thanks!